### PR TITLE
interpreter: mayExit return boolean instead of Exit(0)

### PR DESCRIPTION
**Pull request by coocood**
Fixes issue #54

### CODE REVIEW

**Comment by ngaut** [comment]
Good catch.

File: interpreter/main.go [hunk 1]
```diff
@@ -108,15 +108,17 @@ func executeLine(tx *sql.Tx, txnLine string) error {
 	return nil
 }
 
-func mayExit(err error, line string) {
+func mayExit(err error, l string) bool {
 	if errors2.ErrorEqual(err, liner.ErrPromptAborted) || errors2.ErrorEqual(err, io.EOF) {
 		fmt.Println("\nBye")
 		saveHistory()
-		os.Exit(0)
+		line.Close()
```
Comment (qiuyesuifeng) [inline interpreter/main.go]: no need to use line.Close() here?

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**